docs/deployment/dns.md

```diff
@@ -0,0 +1,68 @@
+- *.<DOMAIN> / Tenant Gateway
+- *.admin.<DOMAIN> / Admin Gateway
+
+{{% alert-note %}}
+The default value for `DOMAIN` is `uds.dev`, which is intended for development purposes only. For non-development purposes, you should override this value by specifying a value for `domain` in your `uds-config.yaml`. You can read more about overriding variables [here](https://uds.defenseunicorns.com/cli/quickstart-and-usage/#variables-and-configuration). 
+{{% /alert-note %}}
+
+You can find each Gateway resource in a dedicated namespace:
+```cli
+$ kubectl get gateway -A
+NAMESPACE              NAME             AGE
+istio-admin-gateway    admin-gateway    1h
+istio-tenant-gateway   tenant-gateway   1h
+```
+
+### Bundle Configuration
+{{% alert-note %}}
+UDS Core does not include any cloud provider specific configuration by default. Additional overrides are required to deploy UDS Core on a given provider. This section will refer to AWS, but values can be substituted as needed for other providers.
+{{% /alert-note %}}
+
+The Admin and Tenant Gateways will be each be bound to an external LoadBalancer that is exposed on TCP ports 80 and 443 by default. The Admin Gateway should be configured to use an internal facing LoadBalancer and the Tenant Gateway should be configured to use an external facing LoadBalancer. Below is an example of overrides that would accomplish this:
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-lb-config
+  description: A UDS example bundle for deploying UDS Core with external Load Balancer configuration
+  version: "0.0.1"
+  
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.27.0-upstream
+
+overrides:
+ istio-admin-gateway:
+   gateway:
+     values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internal"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"
+  istio-tenant-gateway:
+    gateway:
+      values:
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-type
+        value: "external"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-scheme
+        value: "internet-facing"
+      - path: service.annotations.service\.beta\.kubernetes\.io/aws-load-balancer-attributes
+        value: "load_balancing.cross_zone.enabled=true"        
+```
+{{% alert-note %}}
+These service annotations and their values are subject to change. Please reference documentation from your cloud provider to ensure their validity.
+{{% /alert-note %}}
+
+From here, you may create DNS records to associate each LoadBalancer with its appropriate domain name.
```

Comment:
I think it might be helpful to give an example of how to pull the LB CNAME/IPs from the cluster (`k get svc ...` and look at the external ip/host), and then reference setting up a CNAME record for each of the wildcards to point to the host given.